lib/commands/touch.js

```diff
@@ -0,0 +1,124 @@
+import _ from 'lodash';
+import androidHelpers from '../android-helpers';
+import { asyncmap } from 'asyncbox';
+
+let commands = {}, helpers = {}, extensions = {};
+
+helpers.parseTouch = async function (gestures, multi) {
```

Comment:
Not exactly same, we are proxying the request to UiAutomator2 server here, where as in appium-android-driver we are processing using bootstrap.